packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -514,11 +546,12 @@ const NewXostorForm = decorate([
         }
       },
       async createXostorSr() {
-        const { disksByHost, srDescription, srName, provisioning, replication } = this.state
+        const { disksByHost, ignoreFileSystems, srDescription, srName, provisioning, replication } = this.state
 
         this.state._createdSrUuid = await createXostorSr({
           description: srDescription.trim() === '' ? undefined : srDescription.trim(),
           disksByHost: mapValues(disksByHost, disks => disks.map(disk => formatDiskName(disk.name))),
+          ignoreFileSystems: ignoreFileSystems,
```

Comment:
You can use the shorthand directly : `ignoreFileSystems,` (ESLint should warn you in this case)